### ISSUE TITLE
No option for all findings as warnings

### ISSUE THREAD

**Issue by BartoszKlonowski** [jsx-eslint/eslint-plugin-jsx-a11y#887]
I'm actively using this package, but sometimes my focus is not on the a11y at the moment, so I decide to leave something uncovered, until I fully style it, implement it and cover with accessibility in next PR.
I would like to set the `eslint-plugin-jsx-a11y` to only throw a warning, anything is found.
I was unable to find such flag/option, and turning all to be warning only is not a way to go.

It would be great to have the possibility to switch the plugin to throw warnings only, so that I can continue development and focus on the accessibility right before I release, or just when I decide to.

**Comment by ljharb** [comment]
There generally is no such option; primarily because warnings tend to be pretty useless. You can use a JS config to iterate all the rules, though, and manually set them all to warn.

**Comment by BartoszKlonowski** [comment]
@ljharb Thank you for such a fast response!
>There generally is no such option;

Yes, that is why I posted the issue - this is a kind request for such a feature: `--all-as-warnings` flag (details and naming to be discussed of course).
After all, this would be just a flag one can use, if he/she finds it useful/needed.

>primarily because warnings tend to be pretty useless

Not for everyone, I believe. For me warnings are also something I definitely will handle but having it as warnings makes it an obligatory TODO instead of a blocker. I believe there are others who also don't like to have yellow output of linter for a long time. But I also respect your opinion about this, even if such opinion will decline the feature request of mine.
(This is the idea of GH issues in fact - to discuss things, right?)

> You can use a JS config to iterate all the rules, though, and manually set them all to warn.

That is an option, and another would be to temporary disable (not uninstall) the plugin for linter and enable it back when it's time for covering the a11y of all created widgets.

I'm happy to hear your thoughts about this feature.

**Comment by ljharb** [comment]
eslint plugins have no control over flags; that would be eslint core.

Accessibility is supposed to be a blocker :-) I think it's a much better approach to never implement anything in a non-accessible way.

As for warnings, most people set their editor to not show warnings at all, only errors - I certainly do. The reason I claim they're useless is because anything that doesn't block merging is something that can and will be ignored.

To be honest I think your best option is to avoid including the plugin at all until you're ready to use its rules.

**Comment by BartoszKlonowski** [comment]
> Accessibility is supposed to be a blocker :-) 

You got that right, and that is why I installed the plugin. Thing is that now it reports errors for things that are in the middle of development, but yes
> best option is to avoid including the plugin at all until you're ready to use its rules.

This is what I decided to do until I got all widgets designed and ready to be a11y covered before release.

Thank you for the discussion! If there's no agreement (or possibility) to add this, I will close this issue.

Thanks again 👍
